client/src/reducers/filterIdeasReducer.js

```diff
@@ -0,0 +1,18 @@
+import {
+  FILTER_CATEGORY_SUCCESS,
+  FILTER_CATEGORY_FAILURE
+} from '../actions/actionTypes';
+
+const filterIdeas = (state = [], action) => {
+  const { type, ideas, message } = action;
+  switch (type) {
+    case FILTER_CATEGORY_SUCCESS:
+      return ideas;
+    case FILTER_CATEGORY_FAILURE:
+      return message;
+    default:
+      return state;
+  }
+};
+
+export default filterIdeas;
```

Comment:
Default export name should be the same with file name.
For eg,
```javascript
export default filterIdeas;
```
filename should be `path/to/file/filterIdeas.js`